app/org/sagebionetworks/bridge/models/studies/StudyConsentView.java

```diff
@@ -0,0 +1,50 @@
+package org.sagebionetworks.bridge.models.studies;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+/**
+ * Combines the DynamoDB study consent record with the contents of the consent document, 
+ * stored on S3. As long as you retrieve a study consent through the StudyConsentService, 
+ * it should function as if the document content was persisted with the consent record.
+ */
+public class StudyConsentView {
+
+    private final StudyConsent consent;
+    private final String documentContent;
+    
+    public StudyConsentView(StudyConsent consent, String documentContent) {
+        checkNotNull(consent);
+        checkNotNull(documentContent);
+        this.consent = consent;
+        this.documentContent = documentContent;
+    }
+
```

Comment:
Some of this does not need to be visible via the API, and can be hidden. Also add @JsonIgnore to the getStudyConsent().